libkbfs/rekey_fsm.go

```diff
@@ -435,25 +456,23 @@ func (m *rekeyFSM) loop() {
 
 // Event implements RekeyFSM interface for rekeyFSM.
 func (m *rekeyFSM) Event(event RekeyEvent) {
-	select {
-	case m.reqs <- event:
-		return
-	default:
-	}
-	go func() {
-		// We use a spinning loop here to avoid doing a naked `m.reqs <- event`,
-		// which in racy conditions could cause a panic due to sending to closed
-		// channel. The spinning loop is fine here since reactToEvent
+	for {
+		// We use a spinning loop here to avoid doing a naked `m.reqs <-
+		// event`, which in racy conditions could cause a panic due to sending
+		// to closed channel. The spinning loop is fine here since reactToEvent
```

Comment:
Hmm since `select{}` selects communication that can proceed randomly, it seems this can't guarantee requests won't go in after `m.closedCh` is closed, if `m.reqs` is a buffered channel.